core/modules/simpletest/tests/common.test

```diff
@@ -3447,7 +3450,7 @@ class BackdropGetQueryArrayTestCase extends BackdropWebTestCase {
    * Tests that backdrop_get_query_array() correctly explodes query parameters.
    */
   public function testBackdropGetQueryArray() {
-    $url = "http://my.site.com/somepath?foo=/content/folder[@name='foo']/folder[@name='bar']";
+    $url = "http://my.site.com/some_path?foo=/content/folder[@name='foo']/folder[@name='bar']";
```

Comment:
```suggestion
    $url = "http://my.site.com/some-path?foo=/content/folder[@name='foo']/folder[@name='bar']";
```
Maybe "some-path" since it's part of a URL?

Reply:
Yup, that''l work 👍🏼